src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double topParallelSide;
+    private double bottomParallelSide;
+    private double side;
+
+    public IsoscelesTrapezoid(
+            Color color, double topParallelSide,
+            double bottomParallelSide, double side) {
+        super(color);
+        if (topParallelSide < bottomParallelSide) {
+            this.topParallelSide = topParallelSide;
+            this.bottomParallelSide = bottomParallelSide;
+        } else {
+            this.topParallelSide = bottomParallelSide;
+            this.bottomParallelSide = topParallelSide;
+        }
```

Comment:
If you are confident that your users will always pass the correct values (i.e., topParallelSide is always less than or equal to bottomParallelSide), then you could disable this check. However, it is usually better to be more secure and add this check to avoid incorrect use of the class.

In summary, good catch 👍